config/sync/core.extension.yml

```diff
@@ -15,7 +15,6 @@ module:
   config_translation_po: 0
   content_lock: 0
   crop: 0
-  customerror: 0
```

Comment:
Remember to create a PR for a future release which removes the custom error from the codebase entirely :smile: